packages/quantic/force-app/main/default/lwc/quanticResultPrintableUri/quanticResultPrintableUri.html

```diff
@@ -15,6 +15,11 @@
           <lightning-icon class="slds-m-horizontal_xx-small result-parenturi__separator slds-current-color" icon-name="utility:chevronright" size="xx-small"></lightning-icon>
         </div>
       </template>
+      <template lwc:if={shouldDisplayLink}>
```

Comment:
```suggestion
      <template lwc:if={shouldDisplayPrintableUriLink}>
```

I think this is more representative and clear

Reply:
sure!